src/api/types.ts

```diff
@@ -3,3 +3,33 @@ export interface UploadedFileDto {
   hash: string;
   url: string;
 }
+
+export interface ContentModerationResponse {
```

Comment:
Since this is a type that the user may use, we should include it in the root `types.ts` file.

Reply:
Moving